apps/allocations/app/components/App/App.js

```diff
@@ -82,17 +145,11 @@ const App = () => {
   const PanelContent = panel ? panel.content : null
 
   const Wrap = ({ children }) => (
-    <Main assetsUrl={ASSETS_URL}>
+    <Main assetsUrl={ASSETS_URL} scrollView={false}>
```

Comment:
We don't really need to define `ASSETS_URL` since we are using the default value for it

Reply:
done, tested with frontend only